### PR TITLE
DOMA-1180 Fix date displaying 

### PR DESCRIPTION
**Pull request by SavelevMatthew**
Fixed date-displaying in ticket sharing, added "ago" title on ticket change message, fixed unproper translations

### CODE REVIEW

File: apps/condo/lang/ru.json [hunk 1]
```diff
@@ -96,7 +96,7 @@
   "GoToSignIn": "Перейти на страницу входа",
   "Hide": "Скрыть",
   "Leave": "Покинуть",
-  "LessThanMinute": "Меньше минуты",
+  "LessThanMinute": "Меньше минуты назад",
```
Comment (toplenboren) [inline apps/condo/lang/ru.json:99]: LessThanMinuteAgo ?

File: apps/condo/pages/share.tsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ const Share: IShareProps = ({ date, number, details, id }) => {
     const intl = useIntl()
     const locale = get(LOCALES, intl.locale)
     const localizedDate = locale ? dayjs(date || 0).locale(locale) : dayjs(date || 0)
-    const dateFormatted = localizedDate.format('D MMMM Y')
+    const dateFormatted = localizedDate.format('D MMMM YYYY')
```
Comment (toplenboren) [inline apps/condo/pages/share.tsx:48]: Maybe we need to move this format away to const?

**Review by toplenboren — APPROVED** [review]

File: apps/condo/pages/share.tsx (hunk 2 above)
Comment (SavelevMatthew) [reply apps/condo/pages/share.tsx:48]: Not sure about that, since we don't have unified format yet

**Review by Dimitreee — APPROVED** [review]

File: apps/condo/lang/en.json [hunk 3]
```diff
@@ -96,7 +96,7 @@
   "GoToSignIn": "GoToSignIn",
   "Hide": "Hide",
   "Leave": "Leave",
-  "LessThanMinute": "Less then minute",
+  "LessThanMinuteAgo": "Less then a minute ago",
```
Comment (dkoviazin) [inline apps/condo/lang/en.json:99]: Do we need to override all translates of day.js?

File: apps/condo/domains/ticket/utils/helpers.ts [hunk 4]
```diff
@@ -53,11 +53,11 @@ export const getTicketFormattedLastStatusUpdate = (intl, ticket) => {
     const ticketLastUpdateDate = statusUpdatedAt || createdAt
 
     if (ticketLastUpdateDate) {
-        let duration = dayjs.duration(dayjs().diff(dayjs(ticketLastUpdateDate)))
-        if (duration.asSeconds() < 60) return intl.formatMessage({ id: 'LessThanMinute' })
+        let duration = dayjs.duration(dayjs(ticketLastUpdateDate).diff(dayjs()))
+        if (Math.abs(duration.asSeconds()) < 60) return intl.formatMessage({ id: 'LessThanMinuteAgo' })
```
Comment (dkoviazin) [inline apps/condo/domains/ticket/utils/helpers.ts:57]: And if its less then 3600 - do we need to write LessThenHourAgo?

**Review by pahaz — APPROVED** [review]

**Review by dkoviazin — COMMENTED** [review]
Looks like hacks to replace moment. Maybe we can use locale for date.js, as it's already contains all words for humanize

**Pull request merged**